Dockerfile

```diff
@@ -24,7 +24,6 @@ WORKDIR /usr/src
 RUN apt-get update && apt-get install -y wget
 RUN wget "https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb"
 RUN apt-get install -y $CHROME_DEPS
-RUN dpkg -i google-chrome-stable_current_amd64.deb
```

Comment:
Was this supposed to be dropped here?

Reply:
lol that fails on the m1 :P but should be a in a seperate PR anyways